src/screens/SubTags/SubTags.spec.tsx

```
@@ -16,356 +1,429 @@
+      return translations[str] || str;
+    },
+  }),
+}));
+
+// Mock the InfiniteScroll component
+vi.mock('react-infinite-scroll-component', () => ({
+  default: ({ children, dataLength, next, hasMore, loader }: any) => (
+    <div data-testid="infinite-scroll">
+      {children}
+      {hasMore && (
+        <button onClick={next} data-testid="load-more">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)